recipes/ostrich/build.sh

```diff
@@ -0,0 +1,11 @@
+# Temporarily fix stuff with the GH ostrich repo.
+# (1) Replace "g++" by $(CXX) in the Makefile
+sed -i  -E 's/g\+\+/\$\(CXX\)/' make/makefile
+
+# Compile
+cd make
+make GCC
```

Comment:
Is there a make check/test that we can run here?

Reply:
No, unfortunately there isn't I can raise an issue for it upstream as that would be good practice. Not much I can do about it.

Reply:
The issue to eventually make tests within the package is up. Good for now?